README.md

```diff
@@ -1 +1,178 @@
 # ng-bem-components
+
+An [AngularJS](angularjs.org) `angular-bem` module for using components based on [`i-bem.js`](https://bem.info/libs/bem-core/v2.7.0/desktop/i-bem/) framework.
+See [bem-core](https://github.com/bem/bem-core) and [bem-components](https://github.com/bem/bem-components) for further details.
+
+Module will allow you to render `BEMJSON`, help you to work with BEM mods and event, and provide some integrations of `i-bem` blocks with Angular directives.
+
+## Install
+
+```sh
+bower i ng-bem-components --save
+```
+
+`angular-bem` depends on `i-bem__dom`, `jquery`, `BEMHTML` and `BEMTREE` (optional) from `ymodules` modular system.
+Include `angular-bem.js` in your js bundle or html file in any place where `ymodules` already defined.
+
+## Bootstrap Angular app
+
+You will not able to use `ngApp` directive since dependencies provided asynchronously.
+Require `angular-bem` module from `ymodules` to [bootstrap](https://docs.angularjs.org/api/ng/function/angular.bootstrap) your angular app, instead.
+
+```js
+angular.module('my-app', ['angular-bem', ...]);
+
+modules.require('angular-bem', function(){
```

Comment:
In fact, no. `angular-bem` provides module, not angular itself. It don't needed for bootstrap, but we have to wait until it would be defined in angular.